saw-core/src/Verifier/SAW/Term/Pretty.hs

```diff
@@ -40,17 +42,20 @@ import Data.Foldable (Foldable)
 #endif
 import qualified Data.Foldable as Fold
 import qualified Data.Text as Text
+import qualified Data.Text.Lazy as Text.Lazy
 import qualified Data.Vector as V
 import Numeric (showIntAtBase)
-import Text.PrettyPrint.ANSI.Leijen hiding ((<$>))
-import qualified Text.PrettyPrint.ANSI.Leijen as PPL ((<$>))
+import Prettyprinter hiding (Doc)
+import qualified Prettyprinter (Doc)
+import Prettyprinter.Render.Terminal
 import Text.URI
 
 import Data.IntMap.Strict (IntMap)
 import qualified Data.IntMap.Strict as IntMap
 
 import Verifier.SAW.Term.Functor
 
+type Doc = Prettyprinter.Doc AnsiStyle
```

Comment:
The general suggestion for using prettyprinter annotations is to use an abstract annotation here that is relevant to the data being pretty-printed, and then `reAnnotate` that as appropriate at the presentation layer (e.g. if printing to a color-supporting terminal, re-annotate to `AnsiStyle` there).

For example:

    data SAWTermAnn = PrLocalVar | PrConstant | PrExtCns | ...

    type Doc = Prettyprinter.Doc SAWTermAnn

More on this in an additional comment below...